### PR TITLE
MAINT: refactor some sklearnex examples

### PR DESCRIPTION
**Pull request by samir-nasibli**
### Description 
Just a quick refactoring for some sklearnex examples.
Added more comments, that could be useful for the user.

Development that could be used at https://github.com/intel/scikit-learn-intelex/pull/1624
@ethanglaser please take a look.

- [x] I have reviewed my changes thoroughly before submitting this pull request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes, if necessary (updated in # - _add PR number_)
- [x] The unit tests pass successfully.
- [x] I have run it locally and tested the changes extensively.
- [x] I have resolved any merge conflicts that might occur with the base branch.
- [x] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/intel/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_
- [x] I have added a respective label(s) to PR if I have a permission for that.

### CODE REVIEW

File: examples/sklearnex/random_forest_regressor_dpnp.py [hunk 1]
```diff
@@ -18,13 +18,23 @@
...
+# from sklearnex import patch_sklearn, sklearn_is_patched
+# patch_sklearn()
+## Function that can validate current state of patching
+# sklearn_is_patched()
+## Import estimator
+# from sklearn.ensemble import RandomForestRegressor
+
+# We create GPU SyclQueue and then put data to dpctl tensor using
+# the queue. It allows us to do computation on GPU.
+queue = dpctl.SyclQueue("gpu")
```
Comment (icfaust) [inline examples/sklearnex/random_forest_regressor_dpnp.py:37]: is dpctl imported in this?

File: examples/sklearnex/incremental_pca_dpctl.py [hunk 2]
```diff
@@ -13,14 +13,26 @@
...
 # limitations under the License.
 # ==============================================================================
+
+# sklearnex IncrementalPCA example for GPU offloading with dpctl.tensor:
+#    SKLEARNEX_PREVIEW=YES python ./incremental_pca_dpctl.py
+
 import dpctl
 import dpctl.tensor as dpt
 
+# Just directly import estimator from sklearnex
```
Comment (icfaust) [inline examples/sklearnex/incremental_pca_dpctl.py:23]: I'd try to make the commentary about the importing to the users in a single block of comment, ideally without ```##``` (not sure why that is there. Also I would write it in complete sentences due to the audience.

File: examples/sklearnex/random_forest_regressor_dpnp.py [hunk 3]
```diff
@@ -18,13 +18,23 @@
...
 import dpnp
-import numpy as np
 from sklearn.datasets import make_regression
 from sklearn.model_selection import train_test_split
 
+# Just directly import estimator from sklearnex
 from sklearnex.ensemble import RandomForestRegressor
 
-sycl_device = "gpu:0"
+## Or import estimator via sklearnex's patch mechanism from sklearn
```
Comment (icfaust) [inline examples/sklearnex/random_forest_regressor_dpnp.py:27]: Similar comment as in the other file, rework into sentences, try to make it in a single block.

**Review by icfaust — COMMENTED** [review]
is there are reason why ```##``` is being used? also splitting the comment to several lines later is making it harder to read, I'd give the docs a refactor.  Please check the dpctl import in that one file.

File: examples/sklearnex/incremental_pca_dpctl.py (hunk 2 above)
Comment (samir-nasibli) [reply examples/sklearnex/incremental_pca_dpctl.py:23]: Maybe this is not the best solution.  I wanted to separate comments from the code that was commented
Probably it is worth rethinking how to do it better without double ## sign

**Comment by icfaust** [comment]
Generally with some of those small changes, just ping me and I'll approve it.

File: examples/sklearnex/random_forest_regressor_dpnp.py (hunk 1 above)
Comment (samir-nasibli) [reply examples/sklearnex/random_forest_regressor_dpnp.py:37]: Good catch! Thank you

**Comment by syakov-intel** [comment]
@samir-nasibli what needs to be done to get out of Draft state?

**Comment by samir-nasibli** [comment]
> @samir-nasibli what needs to be done to get out of Draft state?

No review.

**Review by icfaust — APPROVED** [review]
Merge on green CI (don't worry about CondaRecipe, there is an issue with sklearn1.6)
